ops/celo/update-geth.sh

```diff
@@ -37,12 +37,14 @@ docker_search_string="(.*op-geth\@sha256:)(.*)"
 gomod_search_string="^(replace github.com/ethereum/go-ethereum .*=> )github.com/.*/op-geth v.*"
 
 # Check that the searches are each matching a single line
-if [ "$(perl -ne "m|${docker_search_string}| && print" ops-bedrock/l2-op-geth.Dockerfile | wc -l)" != "1" ]; then
-  echo "Failed to find exactly one match for docker search string in ops-bedrock/l2-op-geth.Dockerfile" >&2
-  exit 1
+if [ -d "ops-bedrock" ]; then
```

Comment:
Do we need this? I guess the `ops-bedrock` directory is not coming back.

Reply:
@palango I'm just curious about that. We can remove any steps for `ops-bedrock` directory.

Reply:
There's now an optimism [script](https://github.com/ethereum-optimism/optimism/blob/develop/ops/scripts/update-op-geth.py) that does the updating of the go.mod file, it's probably an improvement on the approach here because it uses `go mod edit` so I think we should use that. I've just been fiddling with it to see if it can work for us, I will raise a PR for that.

Reply:
PR here https://github.com/celo-org/optimism/pull/319